kube_apiserver_metrics/assets/dashboards/overview.json

```
@@ -378,7 +378,7 @@
                   "response_format": "timeseries",
                   "queries": [
                     {
-                      "query": "sum:kube_apiserver.etcd_object_counts{$kube_cluster_name}",
+                      "query": "sum:kube_apiserver.apiserver_storage_objects{$kube_cluster_name}",
```

Shouldn’t it be
```suggestion
                      "query": "sum:kube_apiserver.storage_objects{$kube_cluster_name}",
```
?

According to https://github.com/DataDog/integrations-core/pull/10906/files#diff-910783aeda119db93ebe3e2a1f8958ae460c416e5def9bea8485b013f17f97bfR60, I have the feeling that `apiserver_storage_objects` is the OpenMetric name whereas the Datadog metric name is `storage_objects`.

---

Yes, you're right, sorry about that